src/main/java/nl/tudelft/jpacman/annotations/ParameterizedAssignment.java

```diff
@@ -0,0 +1,8 @@
+package nl.tudelft.jpacman.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.CLASS)
+public @interface ParameterizedAssignment {
```

Comment:
In terms of the genericness of it yes, that would work better, but this is the only assignment it will be ran on this year and i would rather not further confuse the students having to put values in their annotations. Some students haven't seen that yet. (first time will probably be runwith for mockito.)

in regards to why not use @Parameterized, some solutions last year used @parameterized instead of the test case inheritance for the further exercises, which is why I chose not to use that for last years dataset. since this should run before that exercise even exists we could use that indeed.